README.md

```diff
@@ -1,6 +1,6 @@
 # Grafana Mimir
 
-<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo"></p>
+<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo" width="400"></p>
```

Comment:
I think 400px width is still a lot, but in any case we should use max-width and not an explicit width to make it scale fine on small screens (eg. mobile).

```suggestion
<p align="center"><img src="images/logo.png" alt="Grafana Mimir logo" style="max-width: 400px;"></p>
```

Reply:
Does style tag work here? I couldn't find any docs on the internet.

Reply:
Style tag applies to any HTML visual element, including `<img>`.

Reply:
By _here_ I meant on GitHub's markdown :)

`style` tag seems to be stripped out: https://gist.github.com/colega/88fffae30d9d26eefb8a3ebd13041c3f

## `width="20"`

<img src="https://grafana.com/static/img/logos/logo-mimir.svg" width="20">


## `style="max-width: 20px;"`

<img src="https://grafana.com/static/img/logos/logo-mimir.svg" style="max-width: 20px;">

Reply:
BTW notice that even though the first one is `width="20"`, Github automatically adds `style="max-width: 100%"` so it won't break the layout.

Reply:
Ok, let's try!

Reply:
TIL :)